### ISSUE TITLE
[Spike] - Use alternate front-end engine (replace RestServer)

### ISSUE THREAD

**Issue by kjdelisle** [strongloop/loopback-next#795]
blocked by https://github.com/strongloop/loopback-next/issues/753

**DRAFT - Please re-groom**

## Story
Investigate what some of the tradeoffs would be in swapping out our RestServer for an alternate implementation using a technology like Express, Koa, Micro, etc.

## Acceptance Criteria
- Create a new server implementation using one of the above technologies
- Create a new Router that leverages our openapi-spec package to generate routes using one of the routing libraries for the chosen framework (ex. if you selected **koa**, use **koa-router** to build the routes based on the existing api spec(s))

## Questions to Answer
- Can we recycle core logic from our spec library between frameworks if we rewrite our router to use `/foo/:id` instead of `/foo/{id}`?
- What approach would a developer need to take to build a custom server for LoopBack that can still leverage our OpenAPI v3 libs?

**Comment by dhmlau** [comment]
Duplicate of #1071
